{
    "type": "bundle",
    "id": "bundle--c99f8462-655c-449e-a3b0-92063825f957",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--55b98c6e-61bb-4195-9621-941cdf35c5d6",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-05-19T01:24:47.813134Z",
            "modified": "2025-05-19T01:24:47.813134Z",
            "name": "CVE-2025-4894",
            "description": "A vulnerability classified as problematic was found in calmkart Django-sso-server up to 057247929a94ffc358788a37ab99e391379a4d15. This vulnerability affects the function gen_rsa_keys of the file common/crypto.py. The manipulation leads to inadequate encryption strength. The attack can be initiated remotely. The complexity of an attack is rather high. The exploitation appears to be difficult. This product is using a rolling release to provide continious delivery. Therefore, no version details for affected nor updated releases are available.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-4894"
                }
            ]
        }
    ]
}